src/app/routes/dashboard/dashboard.component.ts

```diff
@@ -62,6 +64,12 @@ export class DashboardComponent implements AfterViewInit {
   }
 
   ngAfterViewInit(): void {
+    this.activeRoute.queryParams.subscribe(async (params: any) => {
```

Comment:
When you're using observables you do not need to use async/await. FYI Observables uses Observer pattern. Anyway you can do this:
```
this.activeRoute.queryParams.pipe(
  filter(params: {_oob?: string}) => Boolean(_oob))
  map(params: {_oob?: string}) => params._oob),
  take(1) // just to take it only once and unsubscribe, because we might introduce memory leak.
  // I think it should work, but didn't tested it :) 
).subscribe((oob: string) => this.goToVerifiablePresentation(oob)) 
```